docs/whats-new/2019-11.md

```diff
@@ -0,0 +1,107 @@
+---
+title: ".NET docs what's new (November 2019)"
+description: "What's new in .NET docs in November 2019."
+ms.date: 12/16/2019
+---
```

Comment:
Will there be a Feedback section at the bottom? If yes, do we need an author for this to tag the right people correctly?

Reply:
Good catch @MaximRouiller 

@mairaw Could that be the docs team account? Thoughts?

Reply:
Not sure if we can use team aliases for author. Pick someone and add them to the docfx.json file. 😉